dmoj/checkers/bridged.py

```diff
@@ -35,23 +36,27 @@ def check(
     flags=[],
     cached=True,
     type='default',
+    args=None,
     point_value=None,
     **kwargs
 ) -> CheckerResult:
     executor = get_executor(problem_id, files, flags, lang, compiler_time_limit, cached)
 
     if type not in contrib_modules:
-        raise InternalError('%s is not a valid return code parser' % type)
+        raise InternalError('%s is not a valid contrib module' % type)
+
+    args = args or contrib_modules[type].ContribModule.get_args_string()
 
     with mktemp(judge_input) as input_file, mktemp(process_output) as output_file, mktemp(judge_output) as judge_file:
+        args = shlex.split(
```

Comment:
The fact you are using shlex is disturbing, but I also can't think of a better way, so whatever.